app/src/cambodia/java/org/eyeseetea/malariacare/data/database/datasources/ProgramLocalDataSource.java

```diff
@@ -23,4 +23,19 @@ public Program getUserProgram() {
         Program program = new Program(programDB.getName(), programDB.getUid());
         return program;
     }
+
```

Comment:
I think that program associate to media should be program associated to the user then these methods are not necessary and remove media files is not a ProgramDataSource responsibility